modules/nw-udn-cr.adoc

```diff
@@ -14,6 +14,10 @@ The following procedure creates a user-defined network that is namespace scoped.
 //If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
 //====
 
+.Prerequisites
+
+* You have created a namespace and applied the `k8s.ovn.org/primary-user-defined-network` label.
```

Comment:
You have created a namespace with the `k8s.ovn.org/primary-user-defined-network` label.

"applied" makes it sounds like you could add it after